lambdas/shared/t4_lambda_shared/decorator.py

```diff
@@ -73,7 +75,14 @@ def wrapper(event, _):
                 response_headers.update({
                     'access-control-allow-origin': '*',
                     'access-control-allow-methods': 'OPTIONS,HEAD,GET,POST',
-                    'access-control-allow-headers': request.headers.get('access-control-request-headers', ''),
+                    'access-control-allow-headers': (
+                        request.headers.get('access-control-request-headers', '')
+                    ),
+                    # for preflight checks, not sure we need it for header to work?
+                    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers
+                    'access-control-expose-headers': (
+                        request.headers.get('access-control-request-headers', '')
```

Comment:
Alternatively, is this what we want?
```suggestion
                        f"*, Authorization, {QUILT_INFO_HEADER}"
```

Reply:
this looks good